utils/utils.go

```diff
@@ -401,7 +401,18 @@ func ReplaceOrAppendEnvValues(defaults, overrides []string) []string {
 		parts := strings.SplitN(e, "=", 2)
 		cache[parts[0]] = i
 	}
+
 	for _, value := range overrides {
+		// Values w/o = means they want this env to be removed/absent
+		// Note that this assumes an env var can never have a : in it
```

Comment:
::remove should be ok since : isn't allowed in env names, but either way, I removed it and found a way to do what I wanted w/o a funky value.  :-)